code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -209,7 +158,7 @@
 	if(handling_hal) return //avoid conflict with actual hallucinations
 	handling_hal = 1
 
-	if(client && feral) // largely a copy of handle_hallucinations() without the fake attackers. Unlike hallucinations, only fires once - if they're still feral they'll get hit again anyway.
+	if(client && feral >= 10) // largely a copy of handle_hallucinations() without the fake attackers. Unlike hallucinations, only fires once - if they're still feral they'll get hit again anyway.
 		sleep(rand(200,500)/(feral/10))
```

Comment:
Do not sleep in Life() ever. It sleeps the entire mob controller. When you sleep, the mob controller will bail, and stonedmc master will sigh at poor code and pause the entire controller until your sleep is done.

I cannot merge something that sleeps in Life()

It's best if you come up with a way that doesn't involve sleep, but the alternatives are using spawn, or `set waitfor = 0` which means the proc does above does NOT wait for this proc. Be wary of doing this, because the second you sleep, your proc will return `.` at that exact moment, and there's no potential to return other data (without shenanigans like mutual object references).

Reply:
The handle_feral proc is called under a spawn. Will that still wreck the mob controller?

If so, handle_hallucinations still needs to be fixed as well as it does the exact same thing and is called directly by life() - again, under a spawn(0).